ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -276,6 +276,25 @@ public void selectTransactionsUntilPendingIsEmpty() {
     assertThat(parsedTransactions.get(1)).isEqualTo(transaction1);
   }
 
+  @Test
+  public void shouldNotSelectReplacedTransactionWithSameSenderAndNonceButGreaterGas() {
```

Comment:
nit: can probably ditch everything after ...ReplacedTransaction
```suggestion
  public void shouldNotSelectReplacedTransaction() {
```

Reply:
done